From d400314757a8d5d52bd5722d263bfd5886bb6595 Mon Sep 17 00:00:00 2001
From: Philippe Proulx <eeppeliteloop@gmail.com>
Date: Sat, 29 Oct 2016 13:32:57 -0400
Subject: [PATCH] lttng-ust-elf.c: define NT_GNU_BUILD_ID if not defined

On uClibc, NT_GNU_BUILD_ID is not defined, so we define it
manually in this case.

Signed-off-by: Philippe Proulx <eeppeliteloop@gmail.com>
[Philippe: grabbed from this pull request:
    https://github.com/lttng/lttng-ust/pull/39
]
---
 liblttng-ust/lttng-ust-elf.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/liblttng-ust/lttng-ust-elf.c b/liblttng-ust/lttng-ust-elf.c
index 5f27920..beaa7f3 100644
--- a/liblttng-ust/lttng-ust-elf.c
+++ b/liblttng-ust/lttng-ust-elf.c
@@ -29,6 +29,10 @@

 #define BUF_LEN	4096

+#ifndef NT_GNU_BUILD_ID
+# define NT_GNU_BUILD_ID	3
+#endif
+
 /*
  * Retrieve the nth (where n is the `index` argument) phdr (program
  * header) from the given elf instance.
--
2.9.3

